src/arch/ppc64.cpp

```diff
@@ -129,8 +129,13 @@ std::string name()
 
 std::vector<std::string> invalid_watchpoint_modes()
 {
-  throw std::runtime_error(
-      "Watchpoints are not supported on this architecture");
+  // See PowerISA Book III v3.1, Section 5.4.4 and 9.4
```

Comment:
Should it be section 10.4 instead of 9.4? Am I looking at the wrong spec: https://files.openpower.foundation/s/dAYSdGzTfW4j2r2  ?

Reply:
Yes, it should be 10.4 as per latest. Thanks!